feat(lsp): Arguments to explainError/renderDiagnostic from current line (#431)
mrcjkb/rustaceanvim: README.md

  over error diagnostics (if they have an error code).
  
  ```vim
  :RustLsp explainError {cycle?|current?}
  ```
  ```lua
  vim.cmd.RustLsp('explainError') -- default to 'cycle'
  vim.cmd.RustLsp({ 'explainError', 'cycle' })
  vim.cmd.RustLsp({ 'explainError', 'current' })
  ```

>   If cycle option:
>     Like |vim.diagnostic.goto_next|, |explainError| will cycle diagnostics,
>     starting at the cursor position, until it can find a diagnostic with
>     an error code.
>   If current option
>     Searches for diagnostics only in the current cursor line
>   Defaults to 'explainError cycle', if called without an argument 

Reviewer: ```suggestion
  - If called with `cycle` or no args:
    Like `vim.diagnostic.goto_next`,
    `explainError` will cycle diagnostics,
    starting at the cursor position,
    until it can find a diagnostic with an error code.
    
  - If called with `current`:
    Searches for diagnostics only in the
    current cursor line.
```

  together.
  
  ```vim
  :RustLsp renderDiagnostic {cycle?|current?}
  ```
  ```lua
  vim.cmd.RustLsp('renderDiagnostic') -- defaults to 'cycle'
  vim.cmd.RustLsp({ 'renderDiagnostic', 'cycle' })
  vim.cmd.RustLsp({ 'renderDiagnostic', 'current' })
  ```

>   If 'cycle' option:
>     Like |vim.diagnostic.goto_next|, |renderDiagnostic| will cycle diagnostics,
>     starting at the cursor position, until it can find a diagnostic with
>     rendered data.
>   If 'current' option:
>     Searches for diagnostics only in the current cursor line
>   Defaults to 'renderDiagnostic cycle', if called without an argument 

Reviewer: ```suggestion
  - If called with `cycle` or no args:
    Like `vim.diagnostic.goto_next`,
    `renderDiagnostic` will cycle diagnostics,
    starting at the cursor position,
    until it can find a diagnostic with rendered data.
    
  - If called with `current`:
    Searches for diagnostics only in the
    current cursor line.
```
